### PR TITLE
Change default session name

### LINKED ISSUE #767: Session Cookie Leaks Platform Information
In the session cookie, it leaks information that it is a PHP application: `set-cookie: PHPSESSID=2d4e9f55475f734cb00cb45d6dab14c5; expires=Sun, 10-Mar-2019 21:25:58 GMT; Max-Age=604800; path=/`. The cookie name should be altered to a custom name that is not indicative of a PHP application. This discourages en mass scanners from attempting any known PHP vulnerabilities that may have not yet been patched because they know it's a PHP application.

### PR DESCRIPTION
**Pull request by Isopach**
Resolves #767

### CODE REVIEW

**Comment by svpernova09** [comment]
Thanks for the PR, I've requested a review from Adam and other security experts.

**Comment by heiglandreas** [comment]
And if it's "joind.in" or "joind.in-session". It's not the default but anyone looking at the code will know what it is either way. But this way people at least know what the cookie is there for.

**Comment by Isopach** [comment]
Sorry, I didn't know what's an appropriate name for the session. Please feel free to change it.

File: web/index.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 session_set_cookie_params(60*60*24*7); // One week cookie
 session_cache_limiter(false);
+session_name('tt');
```
Comment (dstockto) [inline web/index.php]:
```suggestion
session_name('joindin');
```

Comment (dstockto) [inline web/index.php]:
```suggestion
session_name('joindin');
```

**Comment by svpernova09** [comment]
Thanks @Isopach and all, we've merged #855

**Pull request closed without merging**